src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidgetComponent.js

```diff
@@ -82,7 +82,7 @@ class ThumbnailGalleryComponent extends React.Component {
           <ReactBnbGallery
             show={this.state.lightboxIsOpen}
             backgroundColor="rgba(22,22,22,.9)"
-            zIndex={1000000} // same value as react-cookie-consent
+            zIndex={1000000} // same value as CookieConsent
```

Comment:
* [ ] FYI if we want to make this comment more helpful, we could point to the CSS wording `.cookie-box` or `cookie box`.